skore/tests/unit/displays/confusion_matrix/test_estimator.py

```diff
@@ -77,40 +75,65 @@ def test_normalization(pyplot, forest_binary_classification_with_train_test):
         X_test=X_test,
         y_test=y_test,
     )
+    display = report.metrics.confusion_matrix()
 
-    display_true = report.metrics.confusion_matrix(normalize="true")
-    display_true.plot()
-    assert np.allclose(display_true.confusion_matrix.sum(axis=1), np.ones(2))
-
-    display_pred = report.metrics.confusion_matrix(normalize="pred")
-    display_pred.plot()
-    assert np.allclose(display_pred.confusion_matrix.sum(axis=0), np.ones(2))
-
-    display_all = report.metrics.confusion_matrix(normalize="all")
-    display_all.plot()
-    assert np.isclose(display_all.confusion_matrix.sum(), 1.0)
+    assert isinstance(display.confusion_matrix, pd.DataFrame)
+    assert display.confusion_matrix.columns.tolist() == [
+        "True label",
+        "Predicted label",
+        "count",
+        "normalized_by_true",
+        "normalized_by_pred",
+        "normalized_by_all",
+    ]
+    n_classes = len(display.display_labels)
+    assert display.confusion_matrix.shape[0] == (n_classes * n_classes)
 
 
-def test_custom_labels(pyplot, forest_binary_classification_with_train_test):
+def test_normalization(pyplot, forest_binary_classification_with_train_test):
+    """Check the behaviour of the `normalize` parameter for frame and plot"""
     estimator, X_train, X_test, y_train, y_test = (
         forest_binary_classification_with_train_test
     )
-    """Check the behaviour when passing custom labels to the confusion matrix display.
-    """
     report = EstimatorReport(
         estimator,
         X_train=X_train,
         y_train=y_train,
         X_test=X_test,
         y_test=y_test,
     )
-    custom_labels = ["Negative", "Positive"]
-    display = report.metrics.confusion_matrix(display_labels=custom_labels)
 
-    assert display.display_labels == custom_labels
-    frame = display.frame()
-    assert list(frame.index) == custom_labels
-    assert list(frame.columns) == custom_labels
+    display_true = report.metrics.confusion_matrix()
+    display_true.plot(normalize="true")
+    assert np.allclose(
```

Comment:
Could you use `np.testing.assert_allclose` instead.

Reply:
You can even have `np.testing.assert_allclose(arr, 1.0)` that would broadcast automatically.